cytominer_eval/tests/test_utils/test_grit_utils.py

```diff
@@ -0,0 +1,54 @@
+import random
+import pytest
+import pathlib
+import tempfile
+import numpy as np
+import pandas as pd
+import pandas.api.types as ptypes
+
+from cytominer_eval.utils.grit_utils import (
+    set_grit_column_info,
+    check_grit_replicate_summary_method,
+)
+
+from cytominer_eval.utils.availability_utils import get_available_grit_summary_methods
+
+random.seed(123)
```

Comment:
To be really honest Greg, I think you should definitely change this to be seed(42) for everything 🤷 :)

Reply:
hahaha, yes, it is the answer to the universe after all. Feel free to fix in your next PR :)